privacy-policy.md

```diff
@@ -0,0 +1,73 @@
+
+
+Contact Information: You may use your contact information, such as your email address or phone number, to customize your account or enable certain account features, for example, for login verification or Slaent via SMS. If you provide us with your phone number, you agree to receive text messages to that number from us. We may use your contact information to send you information about our Services, to market to you, to help prevent spam, fraud, or abuse, and to help others find your account, including through third-party services and client applications. You may use your settings for email and mobile notifications to control notifications you receive from Slaent. You may also unsubscribe from a notification by following the instructions contained within the notification or the instructions on our website. 
+
+
+Additional Information: You may choose to provide us with additional information to help improve and personalize your experience across our Services. If you email us, we may keep your message, email address and contact information to respond to your request. If you connect your account on our Services to your account on another service in order to cross-post between our Services and that service, the other service may send us your registration or profile information on that service and other information that you authorize. This information enables cross-posting, helps us improve the Services.
+
+
+Posts, Profile, and other Public Information: Slaent is primarily designed to help you share information with other users and the world. Most of the information you provide us through Slaent is information you are asking us to make public. You may provide us with profile information to make public on Slaent, such as a short tag, your location, or a picture. Additionally, your public information includes the messages you Post; the metadata provided with Posts, such as when you Posted and the client application you used to Post; information about your account, such as creation time, language, country, and time zone; and the lists you create, Posts you reply to. Slaent broadly and instantly disseminates your public information to a wide range of users, customers, and services, including search engines, developers, and publishers that integrate Slaent content into their services, and organizations such as universities, public health agencies, and market research firms that analyze the information for trends and insights. When you share information or content like photos, videos, and links via the Services, you should think carefully about what you are making public. We may use this information to make inferences. Our default is almost always to make the information you provide through the Services public. If you provide us with profile information that information is public. 
```

Comment:
>and organizations such as universities, public health agencies, and market research firms that analyze the information for trends and insights.

Can omit "public health agencies" and "universities". "Market Research Firms" covers Google Analytics.